src/implementations/twig/components/media-container/media-container.html.twig

```diff
@@ -93,6 +99,10 @@
 
 {# Print the result #}
 
+{% if _expandable is not empty %}
+<div class="ecl-media-container__container">
+{% endif %}
+
 <figure class="{{ _css_class }}"{{ _extra_attributes|raw }}>
 {% if _embedded_media is not empty %}
   {% set _embed_class = _ratio ? "ecl-media-container__media ecl-media-container__media--ratio-" ~ _ratio : "ecl-media-container__media" %}
```

Comment:
nothing has changed here, but couldn't we set the common class ecl-media-container__media even at the top of the template and then add the ratio selector conditionally..?